src/main/java/org/rulez/demokracia/pdengine/FakeADAAssuranceProvider.java

```diff
@@ -0,0 +1,17 @@
+package org.rulez.demokracia.pdengine;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FakeADAAssuranceProvider {
```

Comment:
The right place of a fake is in the test code.

Reply:
Should I put the whole class to the tests?